### PR TITLE
docs(sdk): update comments of _create_job_config

### PR DESCRIPTION
**Pull request by capri-xiyue**
**Description of your changes:**
update comments of _create_job_config

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention). 

- [ ] Do you want this pull request (PR) cherry-picked into the current release branch?

    [Learn more about cherry-picking updates into the release branch](https://github.com/kubeflow/pipelines/blob/master/RELEASE.md#cherry-picking-pull-requests-to-release-branch).

### CODE REVIEW

**Comment by capri-xiyue** [comment]
assign @Bobgy

File: sdk/python/kfp/_client.py [hunk 1]
```diff
@@ -618,8 +618,8 @@ def _create_job_config(self, experiment_id, params, pipeline_package_path, pipel
       params: A dictionary with key (string) as param name and value (string) as param value.
       pipeline_id: The id of a pipeline.
       version_id: The id of a pipeline version. 
-        If both pipeline_id and version_id are specified, pipeline_id will take precendence
-        This will change in a future version, so it is recommended to use version_id by itself.
+        If both pipeline_id and version_id are specified, version_id will take precendence.
+        If only pipeline_id is specified, the default version of this pipeline is used to create the run.
```
Comment (chensun) [inline sdk/python/kfp/_client.py:622]: Looks like there's a repeated comment that needs to be fixed as well:

https://github.com/kubeflow/pipelines/blob/759180537788e01a7906432255ad663c296e9518/sdk/python/kfp/_client.py#L573-L576

Comment (capri-xiyue) [reply sdk/python/kfp/_client.py:622]: done

**Pull request merged**